payjoin/src/core/uri/v2.rs

```diff
@@ -45,7 +45,16 @@ fn ohttp(url: &Url) -> Result<OhttpKeys, ParseOhttpKeysParamError> {
     let value = get_param(url, "OH1")
         .map_err(ParseOhttpKeysParamError::InvalidFragment)?
         .ok_or(ParseOhttpKeysParamError::MissingOhttpKeys)?;
-    OhttpKeys::from_str(value).map_err(ParseOhttpKeysParamError::InvalidOhttpKeys)
+
+    let (hrp, bytes) = crate::bech32::nochecksum::decode(value)
+        .map_err(|_| ParseOhttpKeysParamError::InvalidFormat)?;
+
+    let oh_hrp: Hrp = Hrp::parse("OH").unwrap();
```

Comment:
Is this unwrap intentional?

Reply:
I notice we unwrap in other places as well
https://github.com/payjoin/rust-payjoin/pull/1069/files#diff-ac595e7c86e5c31e38a9ad5b9a01e8502e45d13dc8b781f2b151ad517af85c22R233

But I am not sure what the justification is